build.sh

```diff
@@ -69,14 +69,18 @@ cp -r build/Release/* ../bindings/whisper/
 # Navigate back to the root directory
 cd ../
 
-# Navigate to llama.cpp 
+# Navigate to llama.cpp
 cd llama.cpp
 
 echo "Compiling llama.cpp server"
-LLAMA_BUILD_SERVER=1 make
+rm -rf build-server
+mkdir build-server
+cd build-server
+cmake -DLLAMA_BUILD_SERVER=ON ..
```

Comment:
Here you will also need to take care of where CUDA flag was set or not. See the whisper command for reference

Reply:
> Here you will also need to take care of where CUDA flag was set or not. See the whisper command for reference

CUDA flags? What do you mean?